materialize/resources/resource_connection.go

```diff
@@ -301,7 +318,6 @@ type ConnectionBuilder struct {
 	postgresSSLKey                        string
 	postgresSSLMode                       string
 	postgresAWSPrivateLink                string
-	kafkaBroker                           string
 	kafkaBrokers                          []map[string]interface{}
```

Comment:
Recommend introducing a 
```go
type kafkaBroker struct {
    hostname string
    targetGroupPort int
    whateverElse here
}
```

struct for internal use. It'll be more resilient to typos! Then you only need to make sure that the field name in the schema matches the _one_ place where you construct the `kafkaBroker` struct; everything else gets to use e.g. `kafka.hostname` which will be checked by the Go compiler.

Reply:
Yes! I was planning to do that with the work in #36 but just committed that change